src/test/java/com/google/api/codegen/transformer/go/GoSurfaceNamerTest.java

```diff
@@ -33,32 +33,52 @@
 public class GoSurfaceNamerTest {
   @ClassRule public static TemporaryFolder tempDir = new TemporaryFolder();
 
-  private static Model model;
-  private static ApiConfig apiConfig;
-
-  @BeforeClass
-  public static void setupClass() {
+  @Test
+  public void testClientNamePrefix() {
     TestDataLocator locator = TestDataLocator.create(GoGapicSurfaceTransformerTest.class);
-    model =
+    Model model =
         CodegenTestUtil.readModel(
             locator, tempDir, new String[] {"myproto.proto"}, new String[] {"myproto.yaml"});
 
     ConfigProto configProto =
         CodegenTestUtil.readConfig(
             model.getDiagCollector(), locator, new String[] {"myproto_gapic.yaml"});
 
-    apiConfig = ApiConfig.createApiConfig(model, configProto);
+    ApiConfig apiConfig = ApiConfig.createApiConfig(model, configProto);
 
     if (model.getDiagCollector().hasErrors()) {
       throw new IllegalStateException(model.getDiagCollector().getDiags().toString());
     }
+    GoSurfaceNamer namer = new GoSurfaceNamer(apiConfig.getPackageName());
+    List<Interface> services = model.getSymbolTable().getInterfaces().asList();
+    Truth.assertThat(namer.clientNamePrefix(services.get(0))).isEqualTo(Name.from());
```

Comment:
Because this test depends on several files, I suggest making sure we document clearly what's going on. I think it would be good to 

* assert or check explicitly that getPackageName() returns "gopher" (purely for the benefit of humans reading the code; this check isn't the point of the test)
* and then comment above this line that since the first service name in the proto is "gopher" as well, the prefix should be empty